pagopa/pn-frontend
# Feat/pn 2319 privacy and tos pf

## Short description
Adds Terms of Service and Privacy Policy page in PrivacyTOS.page.tsx for pn-personafisica-webapp, visible at the route /privacy-tos

## List of changes proposed in this pull request
- Adds routing to TOS and Privacy page
- Adds TOS and Privacy page
- Changes links to privacy and tos to pointo to the new route 

## How to test
- Go to the route /privacy-tos and check that two accordions one containing the terms of service and the other the privacy policy.
- Check that links that point to "Termini di Servizio" o "Privacy Policy" bring to the TOS and privacy page
